docs/excel/custom-functions-overview.md

```diff
@@ -11,16 +11,29 @@ Custom functions enable developers to add new functions to Excel by defining tho
 
 [!include[Excel custom functions note](../includes/excel-custom-functions-note.md)]
 
-The following illustration shows an end user inserting a custom function into a cell of an Excel worksheet. The `CONTOSO.ADD42` custom function is designed to add 42 to the pair of numbers that the user specifies as input parameters to the function.
+The following illustration shows how you can call a function you've created with JavaScript or Typescript. In this example, the custom function is a clock, designed by the company Contoso. Entering `=CONTOSO.CLOCK` gives you the present time, which updates itself once every second.
```

Comment:
Consider rewording the first sentence to include a mention of the workbook. "The following illustration shows your workbook calling a function you've created with JavaScript or Typescript."